src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -225,7 +225,20 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
     private void processDml(DmlWrapper dmlWrapper, Map<Id, Address__c> mapParentIdAddr) {
         // we must process dmlWrapper ourselves, so we can extract out the appropriate errors that might occur updating contacts 
         // and parents, and associating those errors with the correct addresses.
-        ERR_Handler.Errors errors = TDTM_TriggerHandler.processDMLWithRollback(dmlWrapper, false);
```

Comment:
This code does its own DML work, so we can remove the processDMLWithRollback method.